wisp/wisp-resource-loader/src/test/kotlin/wisp/resources/ResourceLoaderTest.kt

```diff
@@ -187,22 +187,27 @@ class ResourceLoaderTest {
 
         val resource3 = "filesystem:$tempRoot/data3.txt"
 
+        val resource4 = "filesystem:$tempRoot/data4/data4.txt"
```

Comment:
This test seems to be the exact same as the misk one.

Reply:
Yeah there's some sorely needed deduplication work to be done.